tests/integration/test_extract_audio.py

```diff
@@ -0,0 +1,38 @@
+import subprocess
+import sys
+import time
+
+import pytest
+from nv_ingest_api.util.message_brokers.simple_message_broker import SimpleClient
+from nv_ingest_client.client import Ingestor
+from nv_ingest_client.client import NvIngestClient
+
+
+def test_images_extract_only(
```

Comment:
For anything like this can you decorate with `@pytest.mark.integration`; so that it has to intentionally be run.

e.g. `pytest -m integration ./tests/integration`

Reply:
Added in [74b081a](https://github.com/NVIDIA/nv-ingest/pull/636/commits/74b081af2f8a8862f1a244033141dc60fa17592b)